Assets/Scripts/UI/GlobalHUDController.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class GlobalHUDController : MonoBehaviour
+{
+    [SerializeField]
+    private TMP_Text roundTimer;
+    [SerializeField]
+    private float textOffsetY = 180f;
+
+    void Start()
+    {
+        if (PlayerInputManagerController.Singleton.playerInputs.Count > 2)
+        {
+            roundTimer.GetComponent<RectTransform>().anchoredPosition = Vector2.zero;
+        }
+        else
+        {
+            roundTimer.GetComponent<RectTransform>().anchoredPosition = new Vector2(0, textOffsetY);
```

Comment:
The timer text is not positioned correctly for when we have one or two players:

![image](https://user-images.githubusercontent.com/72146678/204818388-13c4bb10-1ab8-4da8-ac4d-77a0b4d9a416.png)
![image](https://user-images.githubusercontent.com/72146678/204818670-372d47c1-cfff-4d57-b6f1-994a2e52899d.png)

Consider expanding the `Text  (TMP)` component to cover the entire screen, then instead of setting the anchored position, switch between top centered text and middle centered text options in the text itself.

Reply:
The text is displayed correctly with any amount of players as long as the aspect ratio is normal. I will make a commit to make it work with other aspect rations than 16:9 though.